Marlin/Configuration_adv.h

```diff
@@ -3,6 +3,11 @@
 
 #include "Conditionals.h"
 
+//===========================================================================
+//=============================  DEBUG Settings  ============================
+//===========================================================================
+#define DEBUG
```

Comment:
Just a note that this is uncommented, your call on whether or not this is enabled when it's pulled in